blog/content/post/2013-01-15-short-hint-for-all-those-who-use-vim-and-pathogen.md

```diff
@@ -1,16 +1,12 @@
 ---
-ID: 62786
-title: >
-  Short hint for all those who use vim and
-  pathogen
+title:   Short hint for all those who use vim and pathogen
+abstract: Today, after updating my wokspace to latest version I encounter below error during vim running:
```

Comment:
This abstract is too short imo, the post description in the list will be looking strange